files/index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=0.5">
+  <title>HSTS Preload Submission</title>
+  <link rel="stylesheet" href="static/style.css">
+  <script src="static/index.js"></script>
+</head>
+<body class="theme-green">
+
+<div class="content first-content">
+
+
+  <div class="form">
+
+    <form id="domain-form">
+
+      <h2><label for="domain">
+        Enter a domain for the HSTS preload list:
+      </label></h2>
+
+      <input id="domain" type="text" placeholder="example.com" autofocus autocorrect="off" autocapitalize="off" spellcheck="false"></input>
```

Comment:
nit: You might want to wrap these lines at 80 characters. I know go doesn't have a line length limit, but html could have one :)